executor/common_linux.h

```diff
@@ -235,8 +235,6 @@ static int netlink_query_family_id(struct nlmsg* nlmsg, int sock, const char* fa
 		debug("netlink: failed to parse family id for %.*s\n", GENL_NAMSIZ, family_name);
 		return -1;
 	}
-	recv(sock, nlmsg->buf, sizeof(nlmsg->buf), 0); // recv ack
```

Comment:
@jpirko I traced first appearance of this line to your:
https://github.com/google/syzkaller/commit/d40d75d2d787#diff-def271e38e744a7dca0b6e9075acb4f2R483
Is it somehow needed in the devlink code?
Note netlink_send_ext does recv of the reply.
Should it have been done after sending DEVLINK_CMD_RELOAD?